CDCgov/prime-reportstream
# 11181 - Add Dashboard to nav

- Add Dashboard to nav
- If Organizations is Missing Transport then show Daily Data

This PR ...

Receivers who download data (instead of connect through API) are shown the Daily Data page instead of the new Data Dashboard page

Test Steps:
1. Log into RS
2. Select a receiver from settings
3. If receiver is in the following list they will see the Daily Data navigation and be taken to Daily Data. If they are not a receiver in this list then they will see Dashboard and be taken to the Data Dashboard.
ca-sbc-phd
ca-scc-phd
co-routt-county-phd
co-san-juan-basin-phd
fl-hillsborough-phd
oh-ccchd-doh
pa-chester-phd
pa-montgomery-phd
pa-philadelphia-phd
pima-az-phd


## Changes

![Screenshot 2023-09-15 at 4 07 05 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/de35fc6e-14fb-4f3e-a3f4-9df47b63d0cc)
![Screenshot 2023-09-15 at 4 07 21 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/c21a0d94-b856-47d4-aa0e-e5b309564abd)


## Checklist

### Testing

## Linked Issues
- Fixes #11181
